### DISCUSSION TITLE
Implement ability to search among contacts

### DISCUSSION THREAD

**Discussion by WenBobo2** [simple-login/app#1881] [Feature Requests]
If user has a lot of contacts per alias, it will be really useful to have a search function to search for reverse alias.

For now it is too inconvenient because of absence of this feature.

@nguyenkims

**Comment by nguyenkims** [comment]
@WenBobo2 this feature is already available on web right?

**Reply by ghost** [reply]
I think he meant to do it on mobile apps

**Comment by naydichev** [comment]
> @WenBobo2 this feature is already available on web right? 

Yes, it's on web, but not on the iOS app.
